### PR TITLE
Start working with importing in Wiki Suite

### PR DESCRIPTION
**Pull request by ishifoev**
#87 #88

### CODE REVIEW

File: src/run-command.php [hunk 1]
```diff
@@ -54,6 +55,7 @@ function runCommand($context, $command)
         "who-works-when" => 1,
         "update-remote-service" => 2,
         "wiki-api" => 2,
+        "wiki-api-import" => 2,
```
Comment (michielbdejong) [inline src/run-command.php:58]: what's the difference between the `wiki-api` command and the `wiki-api-import` command?

**Comment by michielbdejong** [comment]
Looks good so far! Don't forget to add docs about how to set this all up and use it, I would love to try it out!

**Comment by ishifoev** [comment]
sorry I just working in process it is not finished yet. I am thinking about my next steps. I think it will be wiki-export and wiki-import

**Comment by ishifoev** [comment]
> Looks good so far! Don't forget to add docs about how to set this all up and use it, I would love to try it out!

Yes I will do, for all commands working when I finished I write a full doc, that you can work quickly without problem

**Comment by michielbdejong** [comment]
Only problem when I run:
```sh
php src/cli-single.php wiki-api-export wiki
php src/cli-single.php wiki-api-export wiki
psql prejournal
```

I see:
```sql
prejournal=# select * from movements;
  id   | type_  | fromcomponent | tocomponent |     timestamp_      | amount |             description              
-------+--------+---------------+-------------+---------------------+--------+--------------------------------------
 10563 | worked |           603 |         604 | 2022-04-21 00:00:00 |     60 | Meeting - discuss unique identifiers
 10562 | worked |           603 |         604 | 2022-04-21 00:00:00 |     60 | Meeting - discuss unique identifiers
(2 rows)
```

So the command is not idempotent, data gets duplicated. Do you think you can resolve that by storing the wikisuite-assigned identifiers in the sync table?

**Comment by ishifoev** [comment]
Yes, the duplication is coming because we don't have unique value. But in sync table we can use internal_id like unique, that will not repeat after movement go to sync table?

**Comment by ishifoev** [comment]
I fixed that just ```remote_id```` will be unique for us it is URI, if we have the same movement, but in sync table we handle exception if we have duplication we are give a message.

**Comment by ishifoev** [comment]
But you need before drop tables and recreate them after it you can work

**Comment by ishifoev** [comment]
````
GEN_SQL=1 php schema.php > schema.sql
psql -h localhost -d prejournal -U your_username -f schema.sql
````

**Pull request merged**